sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -398,6 +425,10 @@ AZ_NODISCARD az_result az_iot_hub_client_methods_parse_received_topic(
  * @param[in] mqtt_topic_size The size, in bytes of \p mqtt_topic.
  * @param[out] out_mqtt_topic_length __[nullable]__ Contains the string length, in bytes, of \p
  * mqtt_topic. Can be `NULL`.
+ * @pre \p client must not be `NULL`.
+ * @pre \p request_id must be a valid span of size greater than 0.
+ * @pre \p mqtt_topic must not be `NULL`.
+ * @pre \p mqtt_topic_size must be greater than 0.
```

Comment:
Is this the correct precondition in source, should it be `_az_PRECONDITION(mqtt_topic_size > 0);`?
https://github.com/Azure/azure-sdk-for-c/blob/4b2f787bc155d1cd67f64ad1eebfdbc1438f6ce5/sdk/src/azure/iot/az_iot_hub_client_methods.c#L91

Reply:
fixed https://github.com/Azure/azure-sdk-for-c/pull/1621